### DISCUSSION TITLE
Custom Bloody Points texture

### DISCUSSION THREAD

**Discussion by DerpedCrusader** [nzp-team/nzportable#801] [Other Content Releases]
I really like the blood texture for the points, and I didn't want to just, rip a texture, so I made my own! tried to match the red of the round tally mark

![image](https://github.com/nzp-team/nzportable/assets/13917665/fa674d4a-58a6-4845-a726-43b0152d137f)

# [DOWNLOAD](https://archive.org/download/points_202406/Points.zip)

**Comment by Jeyphr** [comment]
I like it. It's so much clearer than the black and white splotch used in the base game.

**Reply by DerpedCrusader** [reply]
Did you notice there's a skull on the left side of the default points counter?
